src/toil/test/wdl/wdltoil_test.py

```diff
@@ -1121,122 +1120,7 @@ def make_string_expr(self, to_parse: str) -> WDL.Expr.String:
         for i in range(1, len(parts), 2):
             parts[i] = WDL.Expr.Placeholder(pos, {}, WDL.Expr.Null(pos))
 
-        return WDL.Expr.String(pos, parts)
-
-    def test_remove_common_leading_whitespace(self) -> None:
```

Comment:
Since we're removing these test that exercise a lot of edge cases, are we sure we have enough tests for the overall behavior (i.e. in workflows in the conformance tests or in Toil) that we'll catch breakage if MiniWDL changes its behavior and stops following the spec again?